.doc_gen/metadata/sns_metadata.yaml

```diff
@@ -786,6 +793,22 @@ sns_Publish:
                 - sns.rust.sns-hello-world
   services:
     sns: {Publish}
+sns_ManageTopicSubscription:
+  title: Manage &SNS; topic subscription using an &AWS; SDK
+  title_abbrev: Manage topic subscription
+  synopsis: manage topic subscription using &SNS;.
+  category:
+  languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/SNS
+          excerpts:
+            - snippet_tags:
+                - SNS.dotnetv3.TopicSubscribe
+                - SNS.dotnetv3.UnsubscribeTopic
+  services:
+    sns: {Subscribe, Unsubscribe}
```

Comment:
I would break this into two and put them under the existing "Subscribe and email address to a topic" and "Delete a subscription" examples.

Reply:
Good suggestion, thank you. Fixed.